Source/Scene/Model.js

```diff
@@ -3387,7 +3436,10 @@ define([
 
             uniformMap = combine(uniformMap, {
                 gltf_color : createColorFunction(model),
-                gltf_colorBlend : createColorBlendFunction(model)
+                gltf_colorBlend : createColorBlendFunction(model),
+                gltf_numClippingPlanes: createNumClippingPlanesFunction(model),
+                gltf_clipNormals: createClippingNormalsFunction(model),
+                gltf_clipPositions: createClippingPositionsFunction(model)
```

Comment:
I changed it to this and used `dot(clipNormal, (positionWC.xyz - clipPosition)`, which cleared up a lot, but not all of the artifacting. I'll take another look once it's in eyespace.

Reply:
If the eye-space plane approach works, we should go with that instead of eye space normals + positions. This would avoid the need to send two sets of arrays over to the shader.